src/components/App.js

```diff
@@ -42,10 +42,10 @@ class App extends Component {
 
   render = () => {
     let currentView;
-    if (this.state.selectedMovie) {
+    if (Object.keys(this.state.selectedMovie).length > 0) {
```

Comment:
Nice job figuring this logic out Chez. Who knew this would take us 30 minutes to figure out?!